src/main.js

```diff
@@ -20,13 +19,25 @@ axios.interceptors.response.use(
   },
   error => {
     if (!process.env.VUE_APP_API_KEY) {
-      alert(
-        "Please make a file named '.env.local'. Create a global variable VUE_APP_API_KEY and initialize it with your own key in this file. " +
-          error
-      );
+      console.log(error);
+      router.replace({
+        name: "Error",
+        params: { id: 1 }
+      });
     }
     if (process.env.VUE_APP_API_KEY && error.response.status === 403) {
-      alert("Your key expired or is invalid  " + error);
+      console.log(error);
+      router.replace({
+        name: "Error",
+        params: { id: error.response.status }
+      });
+    }
+    if (process.env.VUE_APP_API_KEY && error.response.status === 404) {
+      console.log(error);
+      router.replace({
+        name: "Error",
+        params: { id: error.response.status }
+      });
```

Comment:
The only difference between the bodies of these three `if`s is that in the first one, status is replaced by a constant `1`. Seems like it could be factored out or at least simplified.